app/controllers/company.controller.js

```diff
@@ -0,0 +1,130 @@
+const db = require('../models');
+
+const {
+  Company,
+  User,
+  Sequelize: { Op },
+} = db;
+
+// Create a company
+exports.createCompany = async (req, res, next) => {
+  const {
+    body: {
+      userId,
+      companyName,
+      website,
+      primaryColor,
+      secondaryColor,
+      address,
+      logo,
+      bio,
+    },
+  } = req;
+
+  User.findById(userId)
+    .then((user) => {
+      user.update({ isCompanyOwner: true });
```

Comment:
updating the user is a query that takes time. you should add a link in the promise chain for it and then go on